com.gopeed.Gopeed.yml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gopeed
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
```

Comment:
Why do you need full home? Does it support filechoosers?

Reply:
Yes, it uses file_picker to help the user select the directory.

Reply:
> Yes, it uses file_picker to help the user select the directory.

Does it support https://flatpak.github.io/xdg-desktop-portal/docs/#gdbus-org.freedesktop.portal.FileChooser? If so then you wouldn't need this

Reply:
I'm not sure, it use this package: https://pub.dev/packages/file_picker, I'm guessing it's not.

Reply:
> I'm not sure, it use this package: https://pub.dev/packages/file_picker, I'm guessing it's not.

Looks like no https://github.com/miguelpruivo/flutter_file_picker/pull/1275